conf/PROD.public.conf

```diff
@@ -16,3 +16,22 @@ googleAuth.redirectUrl = "https://support.theguardian.com/oauth2callback"
 paymentApi.url="https://payment.guardianapis.com"
 membersDataService.api.url="https://members-data-api.theguardian.com"
 play.filters.headers.contentSecurityPolicy = "default-src 'self' polyfill.guim.co.uk www.paypalobjects.com www.paypal.com www.sandbox.paypal.com js.stripe.com pasteup.guim.co.uk ophan.theguardian.com j.ophan.co.uk media.guim.co.uk www.google-analytics.com www.googletagmanager.com tagmanager.google.com www.googleadservices.com googleads.g.doubleclick.net www.google.com www.google.co.uk optimize.google.com static.ads-twitter.com bat.bing.com bid.g.doubleclick.net t.co analytics.twitter.com stats.g.doubleclick.net www.youtube.com connect.facebook.net www.facebook.com consumer.krxd.net cdn.krxd.net beacon.krxd.net secure.adnxs.com checkout.stripe.com fonts.googleapis.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com sentry.io data: wss: 'unsafe-inline' q.stripe.com payment.guardianapis.com"
+
+// ------------------------ Feature switches ----------------------------
+// To disable a feature change the value from On to Off and redeploy the app.
+// To override a value for local development change it in your private conf file
+// /etc/gu/support-frontend.private.conf and that value will be used instead.
+switches {
+  //Payment methods for one-off contributions
+  oneOff {
+    stripe=On
+    payPal=On
```

Comment:
Insert 24/7 comment here.

Reply:
I feel like Rupert's comment (above) already covers this?

Reply:
For brevity, perhaps we could drop the comment about local development here though, since this is the PROD config

Reply:
I am tempted to say yes, but @johnduffell pointed out before that the 24/7 comment should be more explicit. Therefore, I switched to 24-7 prefix, and also adding the `off` line, in that way 24/7 needs only to comment and uncomment lines of code in case of an emergency.

Reply:
I've made the instructions a bit more explicit and removed the local dev instructions from the Prod file